pyproject.toml

```diff
@@ -1,7 +1,88 @@
+[tool.poetry]
+name = "mindsdb"
+version = "0.0.0"  # leave as 0, it will auto update when pushed to artifact store based on latest git
```

Comment:
To understand a bit bettter - this means we should never update this line? Or does it mean that poetry will automatically change this line to match the latest git?

Reply:
yes correct, it would in place look for a git tag. This can be set up in CI/CD. I already have some example actions from past projects I've worked on